module/zfs/vdev.c

```diff
@@ -1157,22 +1156,20 @@ vdev_open_children(vdev_t *vd)
 	 * in a single thread so that the same thread holds the
 	 * spa_namespace_lock
```

Comment:
The comment probably should be updated as it is not clear why serial execution is used also in case the thread is a member of system_taskq.